R/pkg/R/functions.R

```diff
@@ -3048,6 +3048,21 @@ setMethod("array_position",
             column(jc)
           })
 
+#' @details
+#' \code{array_repeat}: Creates an array containing the left argument repeated the number of times
+#' given by the right argument.
+#'
+#' @param n Column determining the number of repetitions.
+#' @rdname column_collection_functions
+#' @aliases array_repeat array_repeat,Column-method
+#' @note array_repeat since 2.4.0
+setMethod("array_repeat",
+          signature(x = "Column", n = "Column"),
```

Comment:
Scala has this as "col" and "count" so how about either "x" and "count" or "x" and "value" (value already is in used in column_collection_functions)